Tools/WinMLRunner/src/OutputHelper.h

```diff
@@ -136,6 +140,35 @@ class OutputHelper
         }
     }
 
+    void PrintLearningModelDevice(DeviceType deviceType, LearningModelDevice& device)
```

Comment:
const LearningModelDevice&

Reply:
Yep, that makes sense to make it const because the intention isn't to change LearningModelDevice